rpcs3/Emu/RSX/Program/ProgramStateCache.cpp

```diff
@@ -427,13 +427,13 @@ vertex_program_utils::vertex_program_metadata vertex_program_utils::analyse_vert
 
 usz vertex_program_storage_hash::operator()(const RSXVertexProgram &program) const
 {
-	const usz ucode_hash = vertex_program_utils::get_vertex_program_ucode_hash(program);
+	const usz ucode_hash = program.get_sub_hash();
```

Comment:
Storae hash is used to fingerprint ucode dumps on disk, cannot use subhash here.